UPGRADING

```diff
@@ -57,10 +49,6 @@ PHP 8.4 UPGRADE NOTES
     should be replaced with checks for `false`.
 
 - DOM:
-  . Added DOMNode::compareDocumentPosition() and DOMNode::DOCUMENT_POSITION_*
```

Comment:
This was added because last time some stuff got added, I got a complaint that adding the $id property breaks BC because someone was having a user class that implemented $id (differently).

Reply:
> This was added because last time some stuff got added, I got a complaint that adding the $id property breaks BC because someone was having a user class that implemented $id (differently).

Okay, I'll make a note to add a generic warning about this in the incompatibility page.